dlib/media/ffmpeg_abstract.h

```diff
@@ -29,6 +29,22 @@ namespace dlib
                 - Returns a string description of a channel layout, where layout is e.g. AV_CH_LAYOUT_STEREO
         !*/
 
+// ---------------------------------------------------------------------------------------------------
+
+        dlib::logger& logger_internal();
```

Comment:
```suggestion
        dlib::logger& logger_ffmpeg();
```

Reply:
Update docs

Reply:
I'm working on a PR at the moment which re-organises the files. I'm finding it annoying having all the docs in a separate document and having to synchronize everything. I'll commit tonight. It will also include the API changes to `decoder` and `demuxer`. I think it will be cleaner.